### PR TITLE
R4 procedure search by date charted

### PR DESCRIPTION
**Pull request by PavloShabatin**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Before:
image
After:
Screenshot 2021-06-10 at 16 02 11

### CODE REVIEW

File: content/millennium/r4/general-clinical/procedure.md [hunk 1]
```diff
@@ -46,10 +46,12 @@ Search for Procedures that meet supplied query parameters:
  `_id`             | This or `patient` or `subject` | [`token`]     | The logical resource id associated with the resource.
  `patient`         | This or `_id` or `subject`     | [`reference`] | Who the procedure is for. Example: `12345`
  `subject`         | This or `_id` or `patient`     | [`reference`] | Who the procedure is for. Example: `Patient/12345`
+ `date`            | N                              | [`date`]      | Date range into which the procedure falls. Example: `date=gt2015-09-24T12:00:00.000Z&date=le2020-07-15T16:00:00.000Z`
 
 Notes:
 
 * If `_id` is provided, no other parameters may be provided.
+* The `date` parameter must be provided two times, and must use the `ge`, `gt`, `le`, or `lt` prefixes. The lower value must have a `ge` or `gt` prefix and the higher value must have an `le` or `lt` prefix.
```
Comment (Sanvenky) [inline content/millennium/r4/general-clinical/procedure.md:54]: The `date` parameter must be provided two times with time,

File: content/millennium/r4/general-clinical/procedure.md [hunk 2]
```diff
@@ -46,10 +46,12 @@ Search for Procedures that meet supplied query parameters:
  `_id`             | This or `patient` or `subject` | [`token`]     | The logical resource id associated with the resource.
  `patient`         | This or `_id` or `subject`     | [`reference`] | Who the procedure is for. Example: `12345`
  `subject`         | This or `_id` or `patient`     | [`reference`] | Who the procedure is for. Example: `Patient/12345`
+ `date`            | N                              | [`date`]      | Date range into which the procedure falls. Example: `date=gt2015-09-24T12:00:00.000Z&date=le2020-07-15T16:00:00.000Z`
```
Comment (Sanvenky) [inline content/millennium/r4/general-clinical/procedure.md:49]: [`dateTime`]

Comment (Sanvenky) [reply content/millennium/r4/general-clinical/procedure.md:49]: like this
https://github.com/cerner/fhir.cerner.com/pull/661/files

**Review by Sanvenky — APPROVED** [review]

**Review by aki-mathur — APPROVED** [review]

**Comment by Zonglin123** [comment]
Screen Shot 2021-06-18 at 12 49 10 PM

**Pull request merged**